README.md

```diff
@@ -56,15 +56,33 @@ Here are some examples of the content and star speakers we have this year:
 | COM235 | Thanks for joining the Student Zone at Build | [Chloe Condon](https://twitter.com/ChloeCondon), [Crazy Aunt Lindsey](https://www.thefablab.com/), [Liam McGregor](www.linkedin.com/in/liamjmcgregor/), [Morgan Dukes](https://www.linkedin.com/in/morgandukes3/), [Swetha Prabakaran](twitter.com/swethapraba) | [Microsoft the Musical](http://aka.ms/musical) |  |  |
 
 ## Try out Minecraft: Education Edition with MakeCode
-You can try out the Minecraft: Education Edition with these usernames:  
+
+You will need an Office 365 Education account to log in to the full version of Minecraft: Education Edition. If you do not have a license, you can still use Minecraft: Education Edition with a limited number of logins.
+
+Enter your [school email address](https://education.minecraft.net/get-started/) to see if your school account is eligible for a trial. If it’s not, we’ll redirect you to a sign-up page.
+
+
+### You can try out the Minecraft: Education Edition with these usernames:  
 mee01@minecrafteducationedition.com  
 mee02@minecrafteducationedition.com  
 mee03@minecrafteducationedition.com  
 ...  
 mee99@minecrafteducationedition.com  
 Pick one, doesn't matter
 Password for all: Build2020  
-**NOTE: THESE MAY NOT BE AVAILABLE PAST BUILD**
+
+### Minecraft Education 
+
+-You can find more details of [Minecraft education](https://education.minecraft.net/)
```

Comment:
Here I would refer to it as Minecraft: Education Edition as was done above

Reply:
Agreed, Thanks @salmanmkc !